### PR TITLE
Add translations for restore backup dialogs

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

Add descriptions and translations for restore backup dialogs.

### Delete backup
CleanShot 2023-10-04 at 09 54 13

### Restore partial backup
CleanShot 2023-10-04 at 09 53 32

### Restore full backup
CleanShot 2023-10-04 at 09 53 12

### Slow download
CleanShot 2023-10-04 at 09 57 21

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -6284,7 +6293,21 @@
       "password_protection": "Password protection",
       "enter_password": "Please enter a password.",
       "passwords_not_matching": "The passwords does not match",
-      "backup_already_running": "A backup or restore is already running, creating a new backup is currently not possible, try again later."
+      "backup_already_running": "A backup or restore is already running, creating a new backup is currently not possible, try again later.",
```
Comment (c0ffeeca7) [inline src/translations/en.json:6296]:
```suggestion
      "backup_already_running": "A backup or restore is already running. Creating a new backup is currently not possible, try again later.",
```

File: src/translations/en.json [hunk 2]
```diff
@@ -6284,7 +6293,21 @@
...
+      "confirm_delete_title": "Delete backup",
+      "confirm_delete_text": "This backup will be permanently deleted and cannot be restored later.",
+      "restore": "Restore",
+      "close": "[%key:ui::common::close%]",
+      "cancel": "[%key:ui::common::cancel%]",
+      "delete": "[%key:ui::common::delete%]",
+      "download": "Download",
+      "more_actions": "More actions",
+      "remote_download_title": "Potential slow download",
+      "remote_download_text": "Downloading backups over the Nabu Casa URL will take some time, it is recomended to use your local URL instead, do you want to continue?"
```
Comment (c0ffeeca7) [inline src/translations/en.json:6310]:
```suggestion
      "remote_download_text": "You are accessing Home Assistant via remote access. Downloading backups over the Nabu Casa URL will take some time. If you are at home, cancel this dialog and enter your local URL, such as 'http://homeassistant.local:8123'"
```

File: src/translations/en.json [hunk 3]
```diff
@@ -6284,7 +6293,21 @@
...
+      "confirm_restore_full_backup_text": "Your entire system will be wiped and the backup will be restored. Depending of the size of the backup, this can take up to 45 min.",
+      "confirm_delete_title": "Delete backup",
+      "confirm_delete_text": "This backup will be permanently deleted and cannot be restored later.",
+      "restore": "Restore",
+      "close": "[%key:ui::common::close%]",
+      "cancel": "[%key:ui::common::cancel%]",
+      "delete": "[%key:ui::common::delete%]",
+      "download": "Download",
+      "more_actions": "More actions",
+      "remote_download_title": "Potential slow download",
```
Comment (c0ffeeca7) [inline src/translations/en.json:6309]:
```suggestion
      "remote_download_title": "Potentially slow download",
```

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @piitaya :+1:

**Review by bramkragten — APPROVED** [review]

**Pull request merged**